content/en/blog/_posts/2023-12-05-My-first-ever-KubeCon-KubeCon+CloudNativeCon-North-America-2023/index.md

```diff
@@ -0,0 +1,178 @@
+
+<p align="center">
+  <img src="pic4.jpg" alt="Priyanka Sharma KubeCon">
+</p>
+</br>
+
+Every KubeCon has more or less the same schedule except for the talks. The talks vary depending on the [CFPs](https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/program/cfp/) submitted. To give a very basic plot of the event, This was the schedule of KubeCon 2023:
+
+<img src="pic5.png" alt="KubeCon Schedule">
+</br>
+
+In case you want to check the full schedule, click [here](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/program/schedule/). You can change the timezone of the schedule according to your local time.
+
+For a beginner, attending KubeCon can be overwhelming. Some might consider leaving tech and open-source. But hold your nerve. There's a place for everyone in the community. As a beginner, I'd suggest carefully examining the schedule for talks aligned with your interests. Keynotes, mainly by CNCF Ambassadors, are crucial for beginners.
+
+<h2>My experiences</h2>
+
+I want to showcase some of the talks I attended at KubeCon North America virtually.
+</br>
+<img src="ss1.png">
+</br>
+**Keynote: Welcome + Opening Remarks - Priyanka Sharma, Executive Director, Cloud Native Computing Foundation**
+</br>
+<img src="ss2.png">
+</br>
+**Keynote: Windy City Whirlwind: Stirring Up the Cloud Native Ecosystem - Taylor Dolezal, Head of Ecosystem, Cloud Native Computing Foundation**
+</br>
+<img src="ss3.png">
+</br>
+**Keynote: Blueprint Banter: Cloud Native Conversations by the Fireside - Taylor Dolezal, Cloud Native Computing Foundation; Alolita Sharma, Apple; Mike Bowen, Blackrock; Cailyn Edwards, Shopify; Mukulika Kapas, Intuit**
+</br>
+<img src="ss4.png">
+</br>
+**Humans of Cloud Native Panel – Finding Your Way in #TeamCloudNative - Mitch Connors, Aviatrix; Miranda Jaramillo, The Trevor Project; Bart Farrell, CNCF Ambassador; Whitney Lee, VMware**
+</br>
+<img src="ss5.png">
+</br>
+**From Non-Tech to CNCF Ambassador: You Can Do It Too! - Julia Furst, Veeam**
+<br/>
+<img src="ss6.png">
+</br>
+**From Classroom to Cluster: Embracing Kubernetes as a Student and Contributing to Its Development - Nitish Kumar, Faeka Ansari & Grace Nguyen, Independent; Meha Bhalodiya, Red Hat; Leonard Pahlke, Liquid Reply**
+</br>
+<img src="ss7.png">
+</br>
+**Learning Kubernetes by Chaos – Breaking a Kubernetes Cluster to Understand the Components - Ricardo Katz, VMware & Anderson Duboc, Google Cloud**
+</br>
+<img src="ss8.png">
+</br>
+**[Livestream Sponsored by Vulk Coop] Cloud Native Telco Day Hosted by CNCF - Half Day Event**
+</br>
+<img src="ss9.png">
+</br>
+**15,000 Minecraft Players Vs One K8s Cluster. Who Wins? - Justin Head, Super League Gaming & Cornelia Davis, Spectro Cloud**
+</br>
+<img src="ss10.png">
+</br>
+**Rise of the Global Cloud Native Community: The Untold Stories - Ashutosh Kumar, Elastic & Rajas Kakodkar, VMware**
+</br>
+<img src="ss11.png">
+</br>
```

Comment:
Please use Markdown - https://www.digitalocean.com/community/tutorials/markdown-markdown-images has an example. Also, who has the copyright?